src/main/java/utils/ArrayUtils.java

```diff
@@ -0,0 +1,24 @@
+package utils;
+
+public class ArrayUtils {
+    public static int[] mergeTwoSortedArrays(int[] firstArray, int[] secondArray) {
+        int i = 0, j = 0, k = 0;
```

Comment:
why is this method in a separate class?

Reply:
I thought I would be able to use this method in the future.